infra/aws/terraform/management-account/delegated-admins.tf

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# https://docs.aws.amazon.com/organizations/latest/userguide/orgs_integrate_services_list.html
+
+resource "aws_organizations_delegated_administrator" "config_multiaccount" {
```

Comment:
You could use a `for_each` here if you want. Might read better.

Reply:
🤔 I'm hesitant about using `for_each` to handle delegated administration. I'm gonna a `TODO`as a follow-up of this PR and we can revisit this.

Reply:
No worries.

I wouldn't use it without having support for [`moved`](https://developer.hashicorp.com/terraform/language/modules/develop/refactoring#moved-block-syntax).